app/views/users/_edit.haml

```diff
@@ -219,3 +219,12 @@
           .btn.btn-danger{id: "close_account", data: {toggle: "modal", target: "#closeAccountModal"}}
             = t(".close_account_text")
           = render "close_account_modal"
+
+    .row
```

Comment:
I would put that above the export/close part, so that that's still the last thing. But at least split it with a `%hr`.